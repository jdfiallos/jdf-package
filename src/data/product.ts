import { Product } from "../interfaces/Product.Interface";

export const product: Product = {
    id: 1,
    title: "Coffee Mug",
    img: "./coffee-mug.png",
    price: 10,
    description: "Coffee Mug",
  };

  