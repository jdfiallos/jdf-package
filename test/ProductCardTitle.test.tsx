import React from 'react';
import renderer from 'react-test-renderer';
import { ProductCard } from '../src/components/ProductCard';
import {product} from '../src/data/product'
import { ProductTitle } from '../src/components/ProductTitle';


describe('it', () => {
  test('render ProductCard', () => {
      const wrapper = renderer.create(
        <ProductCard  product={product}>
          {
            () => {
              return (
                <ProductTitle />
              )
            }
          }
        </ProductCard>
      )

      expect(wrapper).toMatchSnapshot();
  });
});
