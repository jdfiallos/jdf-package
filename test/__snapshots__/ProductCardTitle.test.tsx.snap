// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it render ProductCard 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffee Mug
  </span>
</div>
`;
